idlharness.js

```diff
@@ -69,6 +69,22 @@ function minOverloadLength(overloads) {
     .reduce(function(m, n) { return Math.min(m, n); });
 }
 
+function throwOrReject(operation, fn, obj, args,  message) {
+    if (operation.idlType.generic !== "Promise") {
+        assert_throws(new TypeError(), function() {
+            fn.apply(obj, args);
+        }, message);
+    } else {
+        fn.apply(obj, args).then(
+                function() {
```

Comment:
No need 4 more space indent for this code block, from the coding style of the whole file.